docs/build-aea-programmatically.md

```diff
@@ -74,7 +74,7 @@ ledger_apis = LedgerApis({}, FETCHAI)
 resources = Resources()
 
 # Create an identity
-identity = Identity(name="my_aea", addresses=wallet.addresses, default_address_key=FETCHAI)
+    identity = Identity(name="my_aea", address=wallet.addresses.get(FETCHAI), default_address_key=FETCHAI,)
```

Comment:
Didn't you get "an unexpected indent" exception with this?

Reply:
No, because the code at the end of the file is correct. Though, thanks for spotting :)